pkg/profiles/validator_test.go

```diff
@@ -277,7 +277,7 @@ func makeProfile(opts ...func(*minderv1.Profile)) *minderv1.Profile {
 
 func loadRawRuleTypeDef() (json.RawMessage, error) {
 	// read rule type from disk and set it up as a fixture
-	f, err := os.Open("../../examples/rules-and-profiles/rule-types/github/branch_protection_allow_force_pushes.yaml")
+	f, err := os.Open("../../examples/rules-and-profiles/rule-types/github/branch_protection_allow_fork_syncing.yaml")
```

Comment:
doesn't allowing fork sync also have the same issue? the branch is no longer mandatory.

Reply:
We are not relying on the `branch` field anymore, instead we use the `allow_fork_syncing` required field, i.e. https://github.com/mindersec/minder-rules-and-profiles/blob/5e9722f3472294e3b2cac6ffa115c36acef7aacd/rule-types/github/branch_protection_allow_fork_syncing.yaml#L41

Reply:
d'oh! that makes sense and somehow I missed it. Thanks for clarifying!

Reply:
Which is interesting because now it makes me wonder why this test worked before...